src/routes/AccountRoute/AccountRouteStyled.js

```diff
@@ -10,6 +10,9 @@ border-left: 1px solid var(--color-grey-6);
 @media (max-width: 767px) {
```

Comment:
Let's use our mediaQuery.js from constants. It has to be like this `@media ${device.tablet} ...`

Reply:
updated accordingly.